services/test_analytics/ta_cache_rollups.py

```diff
@@ -36,32 +36,32 @@ def rollup_blob_path(repoid: int, branch: str | None = None) -> str:
 # so from the time you deprecate an old schema, you only have to keep handling it
 # for 60 days
 NO_VERSION_POLARS_SCHEMA = [
-    "computed_name",
+    ("computed_name", pl.Utf8),
     ("flags", pl.List(pl.String)),
-    "failing_commits",
-    "last_duration",
-    "avg_duration",
-    "pass_count",
-    "fail_count",
-    "flaky_fail_count",
-    "skip_count",
+    ("failing_commits", pl.Int64),
+    ("last_duration", pl.Float64),
+    ("avg_duration", pl.Float64),
+    ("pass_count", pl.Int64),
+    ("fail_count", pl.Int64),
+    ("flaky_fail_count", pl.Int64),
+    ("skip_count", pl.Int64),
     ("updated_at", pl.Datetime(time_zone=UTC)),
-    "timestamp_bin",
+    ("timestamp_bin", pl.Date()),
 ]
 
 V1_POLARS_SCHEMA = [
-    "computed_name",
-    "testsuite",
-    ("flags", pl.List(pl.String)),
-    "failing_commits",
-    "last_duration",
-    "avg_duration",
-    "pass_count",
-    "fail_count",
-    "flaky_fail_count",
-    "skip_count",
+    ("computed_name", pl.Utf8),
+    ("testsuite", pl.Utf8),
+    ("flags", pl.List(pl.Utf8)),
```

Comment:
It is still defined as `pl.String` in the above "no version" schema.
What is the difference between these two anyway?